libc/test/UnitTest/LibcTestMain.cpp

```diff
@@ -31,6 +31,8 @@ TestOptions parseOptions(int argc, char **argv) {
       Options.PrintColor = false;
     else if (arg == "--gtest_print_time")
       Options.TimeInMs = true;
+    else if (arg == "--gtest_brief=1")
+      Options.Brief = true;
```

Comment:
Note: gtest also support the `GTEST_BRIEF` env var. To make this cleaner in the Cmake files, perhaps I should only support the env var, and not the command line flag?